processes-webapp/src/main/webapp/WEB-INF/notification/templates/mail/RequestCommentPlugin.gtmpl

```diff
@@ -0,0 +1,153 @@
+<%
+/*
+* Copyright (C) 2022 eXo Platform SAS.
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Affero General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+%>
+<style>
+    a{
+        color: #2f5e92;
+        text-decoration: none;
+        font-family: 'HelveticaNeue Bold', Helvetica, Arial, HelveticaNeue, sans-serif, serif, tahoma;
+    }
+</style>
+<table border="0" cellpadding="0" cellspacing="0" width="600" bgcolor="#ffffff" align="center" style="background-color: #ffffff; font-size: 13px;color:#333333;line-height: 18px;font-family: HelveticaNeue, Helvetica, Arial, sans-serif;">
+    <tr>
+        <td align="center"  valign="middle" >
+            <table  cellpadding="0" cellspacing="0" width="100%" bgcolor="#ffffff" align="center" style="border:1px solid #d8d8d8;">
+                <tr>
+                    <td  height="45" valign="middle" style="margin:0;height:45px;font-weight:bold;vertical-align:middle;;background-color: #efefef;font-family: 'HelveticaNeue Bold', Helvetica, Arial, sans-serif;color:#2f5e92;font-size:18px;text-align:center">
```

Comment:
@hakermi  @mkrout  it's now possible to reuse headers nd footers in mail  notifs rather than duplicating. Please check how to do it here and apply : https://github.com/Meeds-io/wallet/pull/201/files

Reply:
@plamarque we will apply once we merge on develop.

Reply:
@hakermi please commit changes on the feature branch, ask for QA to validate then merge on develop